test/query/credentialAtomicQueryMTP.test.ts

```diff
@@ -64,7 +64,7 @@ describe("Test claim query NOTIN operation", function () {
             issuerClaimNonRevMtpNoAux: "1",
             claimSchema: "180410020913331409885634153623124536270",
             issuerID: "26599707002460144379092755370384635496563807452878989192352627271768342528",
-            operator: 0,
+            operator: 1,
```

Comment:
The test description is "Test claim query NOTIN operation" but the operator is "equals". It looks like mistake...

Reply:
Changed test description